Collections File Level Annotations/mPowerRerunFileAnnotations.R

```diff
@@ -0,0 +1,869 @@
+                                     'motor coordination'),
+        digitalAssessmentDetails = c('walking (flat)', 
+                                     'standing', 
+                                     'phonation',
+                                     'motor coordination'),
+        dataType=c('analysis'),
+        dataSubtype=c('processed'), 
+        dhPortalIndex = INDEX,
+        dataDescriptionLocation = 'syn23277418/wiki/607321',
+        dataAccessInstructions = 'syn23277418/wiki/607032'),
+    "distance correlation test" = list(
+        consortium = 'mHealth',
+        study = STUDY,
+        studyOrProject = PROJECT_LINK,
+        deviceType = c('handheld'),
+        sensorType = c('accelerometer', 
+                       'gyroscope', 
+                       'magnetometer', 
+                       "microphone",
+                       "touchscreen"),
+        devicePlatform = c('iOS'), 
+        dataCollectionMethod= c('active'),
+        deviceLocation = c('flat surface', 'pocket'),
+        diagnosis= c("Parkinson's disease", 
+                     'control'), 
+        reportedOutcome = c('medication report'),
+        digitalAssessmentCategory= c('gait', 
+                                     'phonation',
+                                     'motor coordination'),
+        digitalAssessmentDetails = c('walking (flat)', 
+                                     'standing', 
+                                     'phonation',
+                                     'motor coordination'),
+        dataType=c('analysis'),
+        dataSubtype=c('processed'), 
+        dhPortalIndex = INDEX,
+        dataDescriptionLocation = 'syn23277418/wiki/607315',
+        dataAccessInstructions = 'syn23277418/wiki/607032')
+)
+
+################################################
+# Figures Annotations
+################################################
+figure_annot_list <- list(
+    "n of 1 analysis" = list(
+        consortium = 'mHealth',
+        study = STUDY,
+        studyOrProject = PROJECT_LINK,
+        deviceType = c('handheld'),
+        sensorType = c('accelerometer', 
+                       'gyroscope', 
+                       'magnetometer', 
+                       "microphone",
+                       "touchscreen"),
+        devicePlatform = c('iOS'), 
+        dataCollectionMethod= c('active'),
+        deviceLocation = c('flat surface', 'pocket'),
+        diagnosis= c("Parkinson's disease", 
+                     'control'), 
+        reportedOutcome = c('medication report'),
+        digitalAssessmentCategory= c('gait', 
+                                     'phonation',
+                                     'motor coordination'),
+        digitalAssessmentDetails = c('walking (flat)', 
+                                     'standing', 
+                                     'phonation',
+                                     'motor coordination'),
+        dataType=c('analysis'),
+        dataSubtype=c(), 
```

Comment:
If not annotated comment/remove this line. Similarly for all the rest of the figure annotations